packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php

```diff
@@ -152,8 +152,15 @@ private function escapeValue($value)
         return Strings::replace($value, '#@#', '@@');
     }
 
-    private function isReservedKey(string $key): bool
+    /**
+     * @param string|int $key
+     */
+    private function isReservedKey($key): bool
     {
+        if (is_int($key)) {
```

Comment:
What about `[false => 'value']`?

Reply:
Ok. I'll change it to `!is_string` tomorrow.